docs/source/command_line.rst

```diff
@@ -342,6 +342,9 @@ Here are some more useful flags:
   when most parts of your program haven't changed since the previous
   mypy run.
 
+  Incremental mode is now the default and may be disabled with
+  ``--no-incremental``.
+
 - ``--cache-dir DIR`` is a companion flag to ``-incremental``, which
```

Comment:
Since incremental mode is the default, this should probably be updated, as people usually won't be passing `--incremental` itself. Also above, the first `--incremental` after the section heading should probably be changed to `Incremental mode`.